### PR TITLE
Peter/fix orjson

### PR DESCRIPTION
**Pull request by probicheaux**
# Description

Reintroduce orjson with fixes for visualizations in responses

Add visualization tests to regression testing

## Type of change

Please delete options that are not relevant.

-   [ ] Bug fix (non-breaking change which fixes an issue)
-   [x] New feature (non-breaking change which adds functionality)
-   [ ] This change requires a documentation update

## How has this change been tested, please provide a testcase or example of how you tested the change?

Locally

## Any specific deployment considerations

Non

## Docs

-   [ ] Docs updated? What were the changes:

### CODE REVIEW

File: inference/core/interfaces/http/orjson_utils.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+class ORJSONResponseBytes(ORJSONResponse):
+    def render(self, content: Any) -> bytes:
+        return orjson.dumps(
+            content,
+            default=default,
+            option=orjson.OPT_NON_STR_KEYS | orjson.OPT_SERIALIZE_NUMPY,
+        )
+
+
+def default(obj):
```
Comment (PawelPeczek-Roboflow) [inline inference/core/interfaces/http/orjson_utils.py:19]: please add typing

File: requirements/_requirements.txt [hunk 2]
```diff
@@ -18,3 +18,4 @@ requests-toolbelt>=1.0.0
 wheel>=0.38.1
 setuptools>=65.5.1
 pytest-asyncio<=0.21.1
+orjson
```
Comment (PawelPeczek-Roboflow) [inline requirements/_requirements.txt:21]: maybe a better placement would be requirements.http.txt?

**Review by PawelPeczek-Roboflow — APPROVED** [review]

File: requirements/_requirements.txt (hunk 2 above)
Comment (paulguerrie) [reply requirements/_requirements.txt:21]: Agreed! And can we pin orjson?

**Review by paulguerrie — COMMENTED** [review]
Looking good! Left one comment in addition to Paweł's comments.

**Pull request merged**